samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -185,11 +194,48 @@ Configuration
 *************
 |config|
 
-Configuration options
-=====================
+Disabling key features
+======================
+
```

Comment:
Please add a note that makes it clear that MQTT is only recommended for applications which need to stay connected constantly and that need to transfer data frequently.  So, while individual services in this sample can be disabled, and the rate at which active services report data can be changed, the reader should not take this as a recommendation to use MQTT inappropriately.